pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1193,22 +1202,17 @@ def on_remove_clicked(self, button):
         part_removed = False
         is_multiselection = self._accordion.is_multiselection
         protected_types = platform.boot_stage1_constraint_dict["format_types"]
+
         for selector in self._accordion.selected_items:
             page = self._accordion.page_for_selector(selector)
-            device = selector.device
-            root_name = None
-            if selector.root:
-                root_name = selector.root.name
-            elif page:
-                root_name = page.pageTitle
+            device_name = selector.device_name
+            root_name = selector.root_name or page.page_title
```

Comment:
We are assuming page is not None here with the change, but I suppose it is intended.

Reply:
I think it is a fatal error to have a selector without a page, so we can probably assume that.